manual_test_websocket/package.json

```diff
@@ -13,10 +13,13 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.449.0"
+    "aws-sdk": "^2.465.0"
```

Comment:
`aws-sdk` can probably go into `dev-dependencies` now if it's still needed for the manual test.

Reply:
Can't go into `dev-dependancies` as it's needed by the handlers of the manual tests  (either running on AWS or offline) as opposed to just the tools that are used to test the serverless-offline plugin + handlers (i.e., everything needed to npm run test).

Reply:
still a dev dependency. a dependency is a dependency for a module necessary to run (or: couldn't run without). even if a test requires a dependency it's still a dev-dependency.

Reply:
that's why mocha, chai etc. are dev dependencies as well.

Reply:
in other words: if someone installs this plugin from npm, it would install `aws-sdk` as well, when it's a dependency.

Reply:
It won't install `aws-sdk` when installing serverless-offline from npm because this is `./manual_test_websocket/package.json` and not `./package.json`.

Reply:
ha, never mind. wasn't aware of package.json in the test folder.  😃